src/box/xrow_update_field.h

```diff
@@ -128,10 +170,8 @@ struct xrow_update_arg_splice {
 	/** New content length. */
 	uint32_t paste_length;
 
```

Comment:
This empty line now seems excess. The is just one member after it, not a group of 2.

Reply:
Done.